include/_mgp.hpp

```diff
@@ -236,6 +238,56 @@ inline mgp_type *type_nullable(mgp_type *type) { return MgInvoke<mgp_type *>(mgp
 
 // mgp_graph
 
+inline bool CreateLabelIndex(mgp_graph *graph, const std::string_view label) {
+  return MgInvoke<int>(mgp_create_label_index, graph, std::string(label).c_str());
```

Comment:
Also, my suggestion here is to pass std::string and use on that string `c_str`

Reply:
I can call .data() on string_view that's better